index.bs

```diff
@@ -242,6 +254,11 @@ spec:webidl; type:interface; text:Promise
         "publisher": "IANA"
     },
 
+    "UTR29": {
+        "href": "http://www.unicode.org/reports/tr29/",
+        "title": "UNICODE Text Segmentation",
+        "publisher": "UNICODE Consortium"
+    },
```

Comment:
Here https://github.com/w3c/webauthn/pull/1241/files#r297719677

Same thing?

Reply:
I mean the whole `<pre class=biblio>`, not just this entry.

Reply:
I'm sorry, I don't understand...?

Reply:
:)

There are two of these blocks:

- https://github.com/w3c/webauthn/blob/master/index.bs#L237-L253
- https://github.com/w3c/webauthn/blob/master/index.bs#L6027-L6230

Why is that?

Reply:
doh!  just an oversight me thinks -- I think it was me that created the 2nd `<pre class=biblio>` block. thx for pointing it out explicitly.  o_0

I have fixed it in subsequent commits...